server/paths/assess.ts

```diff
@@ -0,0 +1,11 @@
+import { path } from 'static-path'
+
+const assessApplicationsPath = path('/assess/applications')
```

Comment:
the test coverage is showing this file as having uncovered lines, whereas the other path files have full coverage... I can't work out how they are covered though 🤔

Reply:
ah, thks I missed that -- I will take a look...

Reply:
maybe it's just because it's never called in a unit test somewhere? looks like the others are, and when I put `expect(paths.submittedApplications.show({ id: submittedApplication.id })).toEqual(
        `/assess/applications/${submittedApplication.id}`,
      )` into the controller test it shows as covered... don't think this needs to hold things up. I wonder if we should put `/* istanbul ignore file */` at the top of these files actually, unless there is anything we think needs covering?

Reply:
thank you 🙇 I've added that routing expectation into the controller test as it seems helpful...